app/ui/src/Pages/Project/pages/KG/components/Filters/Filters.tsx

```diff
@@ -1,13 +1,25 @@
-import React from 'react';
+import React, { useState } from 'react';
 import styles from './Filters.module.scss';
+import ScoreFilter from './components/ScoreFilter/ScoreFilter';
 
 type Props = {};
 function Filters({}: Props) {
+  const max = 100;
+  const [scores, setScores] = useState<number[]>([20, 80]);
+  const topScore = max - scores[0];
+  const bottomScore = max - scores[1];
+
+  const handleSliderChange = (
+    _: React.ChangeEvent<{}>,
+    numbers: number | number[]
+  ) => setScores(numbers as number[]);
```

Comment:
I do not like this as we are not properly checking the format, but it is okey as number | number[] is the type the Slider component uses and I did not find a workaround.

Btw, update the state type here too.
```suggestion
  ) => setScores(numbers as [number, number]);
```